include/rumble.h

```diff
@@ -0,0 +1,51 @@
+#ifndef RUMBLE_H
+#define RUMBLE_H
+
+#include "ultra64.h"
+
+#define RUMBLE_MAX_REQUESTS 64
+
+typedef enum {
+    RUMBLE_STATE_CLEAR,
+    RUMBLE_STATE_RUNNING,
+    RUMBLE_STATE_RESET
+} RumbleState;
+
+typedef struct {
+    /* 0x000 */ u8 rumbleEnable[MAXCONTROLLERS];
+    /* 0x004 */ u8 reqStrengths[RUMBLE_MAX_REQUESTS];       // Source strength modulated by distance to the source
+    /* 0x044 */ u8 reqDurations[RUMBLE_MAX_REQUESTS];       // Duration until decreaseRate kicks in
+    /* 0x084 */ u8 reqDecreaseRates[RUMBLE_MAX_REQUESTS];   // Decreases the strength by this much every Vertical Retrace, once the strength hits 0 the request slot is freed
+    /* 0x0C4 */ u8 reqAccumulator[RUMBLE_MAX_REQUESTS];     // Starts at 0, incremented by the strength every Vertical Retrace
```

Comment:
Should this be plural to be consistent with the other arrays?
```suggestion
    /* 0x0C4 */ u8 reqAccumulators[RUMBLE_MAX_REQUESTS];    // Starts at 0, incremented by the strength every Vertical Retrace
```